pkg/performanceprofile/controller/performanceprofile_controller.go

```diff
@@ -595,6 +600,17 @@ func (r *PerformanceProfileReconciler) isMixedCPUsEnabled(object client.Object)
 	return profileutil.IsMixedCPUsEnabled(object.(*performancev2.PerformanceProfile))
 }
 
+func (r *PerformanceProfileReconciler) isTunedDeferredEnabled(object client.Object) bool {
+	if ntoconfig.InHyperShift() {
+		return false
```

Comment:
I know it's still WIP so I might be judging too early. 
We recently aligned all NTO/PAO features to work on Hypershift let's make sure to keep this spirit, unless there's a really good reason why not.
And even if there's, let's add a comment explaining why we cannot support it on Hypershift.

Reply:
yes, good point. Will fix/clarify by the time the WIP tag is lifted.

Reply:
We need to come up with a mechanism to keep this practice then

Reply:
@yanirq For new features we're usually adding e2e tests.
Since the same tests are running on both OCP and HCP they likely to be failed on HCP in case no one took care to add support for the new feature on HCP